src/main/java/frc/robot/commands/HoldArmCommand.java

```diff
@@ -26,24 +32,27 @@ public HoldArmCommand(Arm robotArm) {
 
     @Override
     public void initialize() {
-      ks = 0;
-      kg = -0.075;
+      ks = 0.1;
+      kg = -0.1;
       kv = 0;
       ka = 0;
+      //kp = 0.04;
+      //ki = 0;
+      //kd = 0;
 
       FF = new ArmFeedforward(ks, kg, kv, ka);
+      //PID = new PIDController(kp, ki, kd);
+      
       System.out.println("Command HOLD ARM COMMAND has started");
     }
   
     @Override
     public void execute() {
-      if(GlobalVars.shouldHoldArm){
         double calc = FF.calculate(
```

Comment:
The formatting here is really hard to read, because you have the parameters on the same indentation as the call.  

You are using the ArmFeedForward class, and the calling parameters for calculate() are the position in Radians (measured with 0 as horizontal), and the velocity set point.

https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/controller/ArmFeedforward.html#calculate(double,double)

Your getPosition() call isn't returning degrees, but rather degrees/16.0.  Your arm parallel constant is a degree offset, so it isn't in the same units as getPosition() and will likely be wrong.  Your velocity set point is a target velocity, but is actually the velocity already in radians (not degrees) and then divided by 12 and double-convert to radians?  

Help me to understand what you are going for here.

Reply:
getPosition calls degrees and takes the gearbox ratio (16:1) into account. It is still degrees, but it is divided to get the correct amount of degrees that would otherwise be skewed by the gear ratio. 

As for getVelocity(), this is still a WIP but after discussing with Anshul it was determined that the velocity setpoint was not being used since we already had a kV? I have removed Math.toRadians in the FF.calculate parameter, and set the getVelocity() method return value to radians/second for clarity. I believe this was code that was copied from Armstrong since our FF structure is identical to theirs.